types.go

```diff
@@ -1008,6 +1008,10 @@ func (s *ScnLoadData) Uri() string {
 	return Utf16ToString((*uint16)(unsafe.Pointer(s.uri)))
 }
 
+func (s *ScnLoadData) RequestId() C.HREQUEST {
+	return s.RequestId

```

Comment:
This is returning the function itself. Instead, this should be:
```go
return s.requestId
```

Reply:
Right, I was confused by the wrong comment:
https://github.com/sciter-sdk/go-sciter/blob/97143d145ba550c4c2d5ccf49ceb8ff7a1962f79/types.go#L996-L1005